tools/cli/commands/create.py

```diff
@@ -0,0 +1,375 @@
+created instance. You can disable that behavior by passing in the
+'--no-connect' flag."""
+)
+
+
+_DATALAB_NETWORK = 'datalab-network'
+_DATALAB_NETWORK_DESCRIPTION = 'Network for Google Cloud Datalab instances'
+
+_DATALAB_FIREWALL_RULE = 'datalab-network-allow-ssh'
+_DATALAB_FIREWALL_RULE_DESCRIPTION = 'Allow SSH access to Datalab instances'
+
+_DATALAB_DEFAULT_DISK_SIZE_GB = 200
+_DATALAB_DISK_DESCRIPTION = (
+    'Persistent disk for a Google Cloud Datalab instance')
+
+_DATALAB_STARTUP_SCRIPT = """#!/bin/bash
+
+PERSISTENT_DISK_DEV="/dev/disk/by-id/google-datalab-pd"
+MOUNT_DIR="/mnt/disks/datalab-pd"
+MOUNT_CMD="mount -o discard,defaults ${PERSISTENT_DISK_DEV} ${MOUNT_DIR}"
+
+format_disk() {
+  mkfs.ext4 -F -E lazy_itable_init=0,lazy_journal_init=0,discard ${PERSISTENT_DISK_DEV}
+  ${MOUNT_CMD}
+}
+
+mount_disk() {
+  mkdir -p "${MOUNT_DIR}"
+  ${MOUNT_CMD} || format_disk
+  mount -o discard,defaults /dev/disk/by-id/google-datalab-pd /mnt/disks/datalab-pd || format_disk
+  chmod a+w "${MOUNT_DIR}"
+}
+
+mount_disk
+"""
+
+_DATALAB_CONTAINER_SPEC = """
+apiVersion: v1
+kind: Pod
+metadata:
+  name: 'datalab-server'
+spec:
+  containers:
+    - name: datalab
+      image: gcr.io/cloud-datalab/datalab:local
+      command: ['/datalab/run.sh']
+      imagePullPolicy: IfNotPresent
+      ports:
+        - containerPort: 8080
+          hostPort: 8080
+          hostIP: 127.0.0.1
+      env:
+        - name: DATALAB_ENV
+          value: GCE
+      volumeMounts:
+        - name: home
+          mountPath: /content
+    - name: logger
+      image: gcr.io/google_containers/fluentd-gcp:1.18
+      env:
+        - name: FLUENTD_ARGS
+          value: -q
+      volumeMounts:
+        - name: varlog
+          mountPath: /var/log
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+  volumes:
+    - name: home
+      hostPath:
+        path: /mnt/disks/datalab-pd
+    - name: varlog
+      hostPath:
+        path: /var/log
+    - name: varlibdockercontainers
+      hostPath:
+        path: /var/lib/docker/containers
+"""
+
+
+def flags(parser):
+    """Add command line flags for the `create` subcommand.
+
+    Args:
+      parser: The argparse parser to which to add the flags.
+    """
+    parser.add_argument(
+        'instance',
+        metavar='INSTANCE',
+        help='a name for the newly created instance')
+    parser.add_argument(
+        '--disk_name',
+        dest='disk_name',
+        default=None,
+        help=(
+            'name of the persistent disk used to store notebooks.'
+            '\n\n'
+            'If not specified, this defaults to having a name based\n'
+            'on the instance name.'))
+    parser.add_argument(
+        '--disk_size_gb',
+        type=int,
+        dest='disk_size_gb',
+        default=_DATALAB_DEFAULT_DISK_SIZE_GB,
+        help='size of the persistent disk in GB.')
+
+    connect_group = parser.add_mutually_exclusive_group()
```

Comment:
Similar comment about pick the default, and offer a single flag for opposite behavior.